### PR TITLE
Login, Signup, and Forgot Password

### PR DESCRIPTION
**Pull request by MayankKumar10**
Adding Login, Signup, and Forgot Password In Ecommerce Project

### CODE REVIEW

File: src/components/Login.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          
+        </section>
+          
+          
+        <!-- <div class="formButtonContainer flex-evenly">
+            <div class="AlertDomMainContainer">
+              <input type="submit" class="create-btn ButtonDomContainer descriptionOne primary-button flex-evenly buttonHoverShadow" value="Create Account" />
+          </div>
+        </div> -->
+
```
Comment (ApurvaSawant11) [inline src/components/Login.html:48]: Remove commented code if not needed

File: src/styles/Login.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/*-------------------------Login page(card)---------------*/
+
+.login-check:invalid {
+	border: var(--Mak-colors-red-500) solid 1px;
+	border-radius: 1.5rem;
+}
+/* .login-check:valid {
+	border: var(--Mak-colors-green-500) solid 1px;
+	border-radius: 1.5rem;
+} */
```
Comment (ApurvaSawant11) [inline src/styles/Login.css:10]: Remove this and also check all the files. Remove commented code from all of them

File: src/images/pc-svg-1.svg [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	<path style="fill:#FFFFFF;" d="M356.924,0H256.012v485.792h100.912c5.768,0,10.488-4.72,10.488-10.488V10.488
+		C367.412,4.72,362.692,0,356.924,0z"/>
+</g>
+<rect x="196.684" y="153.04" width="24.24" height="5.336"/>
+<rect x="231.764" y="153.04" width="24.24" height="5.336"/>
+<g>
+</g>
+<g>
+</g>
+<g>
```
Comment (ApurvaSawant11) [inline src/images/pc-svg-1.svg]: Try deleting these extra `g` tag and see if its changing the SVG. If not, then you can delete them.

**Review by ApurvaSawant11 — APPROVED** [review]
Code looks good! 

Just see if you can provide deploy links. It helps in giving better review

File: src/components/Login.html (hunk 1 above)
Comment (MayankKumar10) [reply src/components/Login.html:48]: Okay, I'll do that.

File: src/styles/Login.css (hunk 2 above)
Comment (MayankKumar10) [reply src/styles/Login.css:10]: Okay, I'll do that.

File: src/images/pc-svg-1.svg (hunk 3 above)
Comment (MayankKumar10) [reply src/images/pc-svg-1.svg]: Okay, I'll do that. Thanks, Apurva

File: src/scripts/modal.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+var modal = document.getElementById("myModal");
+
+// Get the button that opens the modal
+var btn = document.getElementById("myBtn");
+
+// Get the <span> element that closes the modal
+var span = document.getElementsByClassName("close")[0];
+
+// When the user clicks the button, open the modal 
+btn.onclick = function() {
```
Comment (vishal804) [inline src/scripts/modal.js]: Just a suggestion That You Can Use ES 6 Standard that is arrow function

File: src/.vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```
Comment (vishal804) [inline src/.vscode/settings.json]: You can add this File in .gitignore file

**Review by vishal804 — APPROVED** [review]

File: src/.vscode/settings.json (hunk 5 above)
Comment (vishal804) [reply src/.vscode/settings.json]: Everything Else Looks Good

File: src/scripts/modal.js (hunk 4 above)
Comment (MayankKumar10) [reply src/scripts/modal.js]: Okay

File: src/.vscode/settings.json (hunk 5 above)
Comment (MayankKumar10) [reply src/.vscode/settings.json]: Okay, I'll do that. Thanks, Vishal

**Pull request merged**